shared/flashingOsToBootMedia/_default.md

```diff
@@ -1,27 +1,16 @@
-Now we have to flash the downloaded `.img` file onto our {{ $device.bootMedia }}. We recommend using [Etcher][etcher-link], a simple, cross platform {{ $device.bootMedia }} writer and validator. Head over to the [Etcher homepage][etcher-link] and get install it, it only takes a few seconds :)
+The next step is to flash the downloaded image onto your {{ $device.bootMedia }} using [**Etcher**][etcher-link], a simple, cross platform {{ $device.bootMedia }} writer and validator. Once you have **Etcher** installed, start it up. To give **Etcher** access to your {{ $device.bootMedia }}, your system may prompt you to grant administrative privileges.
 
-You can of course use any other {{ $device.bootMedia }} writing software you like, some options are:
-* [win32diskimager][win32-disk-imager] for Windows.
-* [piFiller][pifiller-download] for osx.
-* [dd or "Disk Destroyer"][dd-link] for Linux.
```

Comment:
For tidiness, I guess some of these link-URLs can now be removed from the footer-section of this page.